stories/molecules/Dropdown.stories.tsx

```diff
@@ -2,138 +2,91 @@ import * as React from 'react';
-                    <ListItem key="first-key" value="first-value">
-                        ListItem with value
-                    </ListItem>
-                    <Separator />
-                    {['ListItem with value 1', 'ListItem with value 2'].map((value) => (
-                        <ListItem key={value} value={value}>
-                            {value}
-                        </ListItem>
-                    ))}
-
-                    <ListItem key="second-key" value="second-value">
-                        <div style={styles.customListItem}>
-                            <IconTextkernel style={styles.icon} />
-                            <strong>Custom ListItem with value</strong>
-                        </div>
-                    </ListItem>
-                    <div style={styles.customDiv}>Just custom div element</div>
-                </Dropdown>
-            </div>
-        );
-    },
+    render: () => <p>To be implemented</p>,
 };
 
 export const _DropdownWithSingleSelectItem: Story = {
     name: 'Dropdown - Single Select Item',
     args: {},
     render: (args) => {
         const [selectedValue, setSelectedValue] = React.useState<string>();
+        const items = ['Item 1', 'Item 2', 'Some other item', 'Option you like'];
+        const items2 = ['Item 3', 'Item 4'];
+
+        const handleOnChange = (value: string) => {
+            console.log('handleOnChange was called with the following value:', value);
 
-        const onDropdownStateChange = (object) => {
-            console.log(
-                'onDropdownStateChange was called with the following:',
-                JSON.stringify(object)
-            );
+            setSelectedValue(value);
         };
 
         return (
-            <div style={styles.content}>
-                <Dropdown<string>
-                    {...args}
-                    button={<Button context="primary">Click me!</Button>}
-                    onChange={(value) => setSelectedValue(value)}
-                    additionalSelectProps={{
-                        onStateChange: onDropdownStateChange,
-                    }}
-                >
-                    {['Value 1', 'Value 2', 'Value 3'].map((value) => (
+            <DropdownRoot>
+                <DropdownTrigger>
+                    <Button>Apply tags</Button>
+                </DropdownTrigger>
+                <DropdownPortal>
+                    <DropdownContent {...args}>
                         <SingleSelectItem
-                            key={value}
-                            value={value}
-                            isSelected={selectedValue === value}
+                            onSelect={() => {
+                                handleOnChange('1');
+                            }}
+                            key="1"
+                            isSelected={selectedValue === '1'}
                         >
-                            {value}
+                            Select all
```

Comment:
I think this value is confusing in the implementation, looks like something you would have in a MultiSelect thing. Also not sure what it supposed to do. Maybe the implementation is not complete...